docs/hosting.md

```diff
@@ -113,20 +146,33 @@ molstar.Viewer.create('app', {
 })
 ```
 
-# All-in-one solution
-How to build database and run both API and frontend 
-To build database, and to run both frontend and api, from molstar-volseg directory (default) run:
+## Build & Host Script
+
+We use the [build_and_deploy.py](../preprocessor/src/tools/deploy_db/build_and_deploy.py) script to preprocess the database and host the API and Landing Page. Modifying this script might be useful when running the solution on your own data.
```

Comment:
I think it will not host Landing Page

Reply:
The landing page is what you get when you go to https://molstarvolseg.ncbr.muni.cz , not the viewer